src/Core/src/Platform/Windows/WrapperView.cs

```diff
@@ -194,6 +202,12 @@ async Task CreateShadowAsync()
 			double width = _shadowHostSize.Width;
 			double height = _shadowHostSize.Height;
 
+			if (height <= 0 && width <= 0)
+				return;
```

Comment:
I'm a bit curious about this if statement because it was removed in a previous commit (https://github.com/dotnet/maui/commit/50f9024081aa65c6b65bdde20c68bdb4bcd53e16#diff-61362aac9bd0efecc45f0aec6d7c8c6d55758504c003e1109e7b4da0940bbae8L204), but now we are bringing  it back. I think it makes sense to have this check but I just wanted to double check this won't introduce any regressions related to that previous change.

Reply:
Without the `FrameworkElement `loaded and after pass measure, have not sense to create any `DropShadow `because we use the size of the FrameworkElement  to create it.